src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/config/DocumentConfiguration.java

```diff
@@ -186,4 +201,13 @@ private boolean isHighCourtSelected(CaseDetails caseDetails) {
         }
         return false;
     }
+
+    private boolean isHighCourtSelectedV2(FinremCaseDetails caseDetails) {
+        FinremCaseData caseData = caseDetails.getData();
+        return ObjectUtils.isNotEmpty(caseData)
+            && ObjectUtils.isNotEmpty(caseData.getRegionWrapper())
+            && ObjectUtils.isNotEmpty(caseData.getRegionWrapper().getDefaultRegionWrapper())
+            && ObjectUtils.isNotEmpty(caseData.getRegionWrapper().getDefaultRegionWrapper().getHighCourtFrcList())
+            && caseData.getRegionWrapper().getDefaultRegionWrapper().getHighCourtFrcList().getValue().equalsIgnoreCase("highcourt");
+    }
```

Comment:
Please use CCDConfigConstant.HIGHCOURT of hard coded value

Reply:
change applied.